ee/tabby-schema/src/schema/thread.rs

```diff
@@ -50,6 +50,7 @@ pub trait ThreadService: Send + Sync {
     /// Create a new thread run
     async fn create_run(
         &self,
+        user_id: &ID,
```

Comment:
what is this for?

Reply:
pass the user_id to answer service, so that logger could utilize it to log the chat user

Reply:
i recommend just pass `user: &User` (and you can omit user_id and policy).